extractors/pornhub/pornhub.go

```diff
@@ -1,18 +1,28 @@
 package pornhub
 
 import (
-	"bytes"
+	"compress/flate"
+	"compress/gzip"
 	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"regexp"
 	"strings"
 
 	"github.com/iawia002/lux/extractors/types"
 	"github.com/iawia002/lux/request"
 	"github.com/iawia002/lux/utils"
+	"github.com/robertkrimen/otto"
```

Comment:
This should be in a separate group above

Reply:
This is gofmt's default behavior, this ordering is not up to me, I recommend keeping this